### PR TITLE
Epmrpp 74450

### PR DESCRIPTION
**Pull request by Cyberglamdring**
- [x] Added new environments for API, UAT, Jobs, Analyzer

- [x] Separated NodeSelector for API, UAT, Jobs, Analyzer, Analyzer Train, Metrics Gatherer

### CODE REVIEW

File: reportportal/values.yaml [hunk 1]
```diff
@@ -200,6 +210,7 @@ postgresql:
     user: rpuser
     dbName: reportportal
     password:
+    connections:
```
Comment (HardNorth) [inline reportportal/values.yaml:213]: I can't find where you use this.

File: reportportal/values.yaml [hunk 2]
```diff
@@ -101,8 +105,11 @@ servicejobs:
     limits:
       cpu: 100m
       memory: 372Mi
+  jvmArgs:
```
Comment (HardNorth) [inline reportportal/values.yaml:108]: please specify `{}` after colon, to show expected value format.

File: reportportal/values.yaml [hunk 3]
```diff
@@ -36,7 +36,9 @@ uat:
   sessionLiveTime: 86400
   podAnnotations: {}
   jvmArgs: "-Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
+  additionalEnvironments: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:39]: Please add comment on the value, what exactly it does. Ideally our values should look like here: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
(each value commented, examples provided)

File: reportportal/values.yaml [hunk 4]
```diff
@@ -36,7 +36,9 @@ uat:
...
   securityContext: {}
+  nodeSelector: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:41]: Please remove root `nodeSelector` values, since you update it in a way it becomes redundant.

Comment (HardNorth) [inline reportportal/values.yaml:41]: Please add comment on the value

File: reportportal/values.yaml [hunk 5]
```diff
@@ -77,8 +79,10 @@ serviceapi:
   queues:
     totalNumber:
     perPodNumber:
+  additionalEnvironments: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:82]: Please add comment on the value

File: reportportal/values.yaml [hunk 6]
```diff
@@ -77,8 +79,10 @@ serviceapi:
...
   podAnnotations: {}
   securityContext: {}
+  nodeSelector: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:85]: Please add comment on the value

File: reportportal/values.yaml (hunk 2 above)
Comment (HardNorth) [inline reportportal/values.yaml:108]: Please add comment on the value

File: reportportal/values.yaml [hunk 7]
```diff
@@ -101,8 +105,11 @@ servicejobs:
     limits:
       cpu: 100m
       memory: 372Mi
+  jvmArgs:
+  additionalEnvironments: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:109]: Please add comment on the value

File: reportportal/values.yaml [hunk 8]
```diff
@@ -101,8 +105,11 @@ servicejobs:
...
   podAnnotations: {}
   securityContext: {}
+  nodeSelector: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:112]: Please add comment on the value

File: reportportal/values.yaml [hunk 9]
```diff
@@ -141,8 +148,9 @@ serviceanalyzer:
   podAnnotations: {}
   securityContext: {}
   # environment variables for analyzer
-  env: {}
+  additionalEnvironments: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:151]: Please correct comment on the value

File: reportportal/values.yaml [hunk 10]
```diff
@@ -141,8 +148,9 @@ serviceanalyzer:
...
   # ES_CHUNK_NUMBER: 2000
+  nodeSelector: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:153]: Please add comment on the value

File: reportportal/values.yaml [hunk 11]
```diff
@@ -155,6 +163,7 @@ serviceanalyzertrain:
       memory: 512Mi
   podAnnotations: {}
   securityContext: {}
+  nodeSelector: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:166]: Please add comment on the value

File: reportportal/values.yaml [hunk 12]
```diff
@@ -176,6 +185,7 @@ metricsgatherer:
       memory: 256Mi
   podAnnotations: {}
   securityContext: {}
+  nodeSelector: {}
```
Comment (HardNorth) [inline reportportal/values.yaml:188]: Please add comment on the value

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: reportportal/values.yaml (hunk 1 above)
Comment (Cyberglamdring) [reply reportportal/values.yaml:213]: I found this value here: [tempates/api-deployment.yaml](https://github.com/reportportal/kubernetes/blob/develop/reportportal/templates/api-deployment.yaml#L76)

**Review by HardNorth — APPROVED** [review]

**Pull request merged**